mozilla/fx-private-relay
# For MPP-1639: Add RelayAddress.used_on field

This PR is for MPP-1639.

How to test:
1. `python manage.py migrate`
2. http://127.0.0.1:8000/api/v1/relayaddresses/
   * [x] You should see a new `used_on` field
3. Use the django admin site to add some comma-separated domains into the new `used_on` field of an address. (e.g., `disneyplus.com,netflix.com`
4. Go to http://127.0.0.1:8000/api/v1/relayaddresses/?used_on=disney
   * [x] You should see the alias matching "disney" in the list!
5. Go to http://127.0.0.1:8000/api/v1/relayaddresses/?used_on=hulu
   * [x] You should NOT see the alias matching "disney" in the list!
